README.md

```diff
@@ -113,7 +157,7 @@ Your Answer:
 
 Select all that apply:
 ```
-[] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
+[x] Synchronous code runs at an even pace, asynchronous code runs with uneven pacing.
 [] Synchronous code runs all at the same time, asynchronous code runs completely randomly
 [] Synchronous code runs in order (as appears in the source), asynchronous code may run at a later time.
```

Comment:
@jshawl The reason I picked the selection I did was bc I thought of it as the best choice. The reasoning is pretty weird bc I took the definition of asynchronous  they supplied (can run at a later time) as it could run at any time such as a day later. I over thought it. I do understand asynchronous lol I just always get technical with words.